Makefile

```diff
@@ -20,22 +20,38 @@ COVERAGE_ENFORCER_FLAGS=-package github.com/launchdarkly/go-server-sdk/v7 \
 	-skipcode "// COVERAGE" \
 	-packagestats -filestats -showcode
 
-.PHONY: build clean test test-coverage benchmarks benchmark-allocs lint
+ALL_BUILD_TARGETS=sdk ldotel
```

Comment:
Should we add the test service? It does not pass linting right now, because it doesn't seem to have accounted for the nested modules not working with `./...`.

Reply:
I am going with no. We can add it to linting later, but fixing all its lint problems will need a PR as well.